modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -323,7 +323,7 @@ module.exports = {
           underline: [ 'u' ],
           anchor: [ 'span' ],
           superscript: [ 'sup' ],
-          subscript: [ 'sub' ]
+          subscript: [ 'sub' ],
```

Comment:
Would it be possible for you to remove this commit from your PR? Somehow you have some of boutell's commits mixed in with your PR.